example/react-native.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require("path");
+
+if (
+  process.argv.includes("--config=metro.config.windows.js") ||
+  process.argv.includes("autolink-windows") ||
+  process.argv.includes("run-windows")
+) {
+  const sourceDir = "windows";
+  module.exports = {
+    project: {
+      windows: {
+        sourceDir,
+        solutionFile: "Example.sln",
+        project: {
+          projectFile: path.relative(
+            path.join(__dirname, sourceDir),
+            path.join(
+              "node_modules",
+              ".generated",
+              "windows",
+              "ReactTestApp",
+              "ReactTestApp.vcxproj"
+            )
+          ),
+        },
+      },
+    },
+    reactNativePath: "node_modules/react-native-windows",
+  };
+} else {
+  const sourceDir = "android";
+  module.exports = {
+    project: {
+      android: {
+        sourceDir,
+        manifestPath: path.relative(
+          path.join(__dirname, sourceDir),
+          path.join(
+            path.dirname(require.resolve("react-native-test-app/package.json")),
+            "android",
+            "app",
+            "src",
+            "main",
+            "AndroidManifest.xml"
+          )
+        ),
+      },
+      ios: {
+        project: "ios/ReactTestApp-Dummy.xcodeproj",
```

Comment:
Have you tried setting this to:

```suggestion
      project: "node_modules/.generated/ios/ReactTestApp.xcodeproj",
```

`yarn ios` will output an error but start the app anyway:

```
% yarn ios
yarn run v1.22.10
$ react-native run-ios
error Could not find "Podfile.lock" at null.lock. Did you run "pod install" in iOS directory?
error Could not find the following native modules: ReactTestApp-DevSupport. Did you forget to run "pod install" ?
info Found Xcode workspace "MenuExample.xcworkspace"
info Launching iPhone 12 (iOS 14.5)
info Building (using "xcodebuild -workspace MenuExample.xcworkspace -configuration Debug -scheme MenuExample -destination id=55FCD6EA-B6C0-4491-ABE9-074D19045567")
success Successfully built the app
info Installing "~/Library/Developer/Xcode/DerivedData/MenuExample-gjjkxqorvpxgfpamfozsblsaxpie/Build/Products/Debug-iphonesimulator/ReactTestApp.app"
info Launching "com.microsoft.ReactTestApp"
success Successfully launched the app on the simulator
```

I will try to fix the error but it seems like this PR can proceed?

Reply:
(Also very strange that it used to stop after the error when I first tried this a couple of weeks ago.)

Reply:
Never mind. I did things in the wrong order.

Reply:
Just an FYI: I have a fix for the CLI but it's taking a while to get reviewed/merged: https://github.com/react-native-community/cli/pull/1444

Reply:
This should now be fixed in [@react-native-community/cli-platform-ios@5.0.2](https://www.npmjs.com/package/@react-native-community/cli-platform-ios/v/5.0.2).